python-service/app/services/pptx_processor.py

```
@@ -0,0 +1,339 @@
+                    response = client.get(file_path)
+                    response.raise_for_status()
+
+                    # Save to temporary file
+                    temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".pptx")
+                    temp_file.write(response.content)
+                    temp_file.close()
+                    local_pptx_path = temp_file.name
+                    temp_file_path = temp_file.name
+                    logger.info(f"Downloaded PPTX to: {local_pptx_path}")
+            except httpx.TimeoutException:
+                logger.error(f"PPTX download timeout after {DOWNLOAD_TIMEOUT}s")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)